astroquery/mast/cutouts.py

```diff
@@ -56,35 +68,67 @@ def _parse_cutout_size(size):
         either pixels or degrees.
     """
 
+    # This local variable will change to True if input cutout size exceeds recommended limits for TESS
+    limit_reached = False
+
+    # Checking 2d size inputs for the recommended cutout size
+    if not isinstance(size, (int, float, u.Quantity)) & (mission == 'TESS'):
```

Comment:
My goal here was to catch for any array-like data structures, so lists tuples or np arrays. Unfortunately, I can't call for them explicitly because `u.Quantity` objects are arrays themselves, so I went with _excluding_ anything that is a single int/float/u.Quantity value, which will implicitly pick up data structures. 

<img width="324" alt="image" src="https://user-images.githubusercontent.com/32107699/230442602-302fb0f3-a52d-43dc-b859-ef66d273d26b.png">

Reply:
Needed another parenthesis around the `not isinstance(size, (int, float, u.Quantity))` - this has been patched.